bundles/io/org.eclipse.smarthome.io.rest.core/META-INF/MANIFEST.MF

```diff
@@ -50,7 +50,9 @@ Import-Package: com.google.common.base,
  org.osgi.service.cm,
  org.osgi.service.component,
  org.slf4j
-Export-Package: org.eclipse.smarthome.io.rest.core.item,
+Export-Package: org.eclipse.smarthome.io.rest.core,
+ org.eclipse.smarthome.io.rest.core.internal,
```

Comment:
Good catch - I guess this was only a remainder from a previous refactoring of @danchom.
So: Revert or follow-up PR? :-) Note that @danchom has already left for xmas holidays...